### DISCUSSION TITLE
GH Education for homeschool teachers

### DISCUSSION THREAD

**Discussion by b00y0h** [community/community#19224] [GitHub Education]
Hello,

I currently homeschool my three teenage children and I’m trying to apply as an educator. How does one do that as a homeschooling parent?

for context, I recently got declined with the following reason:

> The item you uploaded is insufficient to demonstrate your current academic status. Modifications to your photo proof might have been detected by the system.

even though I chose ‘homeschool’ as my school and applied as a ‘teacher’

Thanks

**Comment by melvincwng** [comment]
@b00y0h,

From this [Github Education webpage](https://education.github.com/discount_requests/teacher_application), it's mentioned that "to qualify for teacher benefits, your must":

- Be an educator, faculty member, or researcher.
- Have a verifiable school-issued email address or upload documents that prove your school affiliation.
- Have a GitHub user account.

While you do indeed have a Github account, and are an educator for your children, I guess the reason for the rejection of your previous application was probably due to the lack of verifiable school-issued email address/documents that prove your school affiliation.

I'm not sure if homeschooling qualifies you for the teacher benefits of GitHub Education. You might want to raise a ticket and check with [GitHub Education Support](https://support.github.com/contact/education) over here.

Hope this clarifies.

**Reply by melvincwng** [reply]
@b00y0h, if this reply has helped you, please kindly 'Mark as answer' so that others are aware that this issue has been addressed/answered.

Thank you, and have a nice day 😄
